### PR TITLE
Issue #7157 Fixed

### PR DESCRIPTION
**Pull request by suraj-webkul**
Issue #7157 fixed

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
Resolve the **conflicts**.

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
**Not working**, and the admin is **not** able to **drop** images.

**Comment by suraj-webkul** [comment]
Working  fine, please verify again

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
Please add a border for the drop area while the admin is trying to drop any image.

File: packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/images.blade.php [hunk 1]
```diff
@@ -159,7 +164,9 @@ class="preview"
                 return {
                     imageData: '',
 
-                    new_image: 0
+                    new_image: 0,
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/images.blade.php:167]: camelCase

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Pull request closed without merging**